sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -547,6 +550,27 @@ def test_implicit_builder_with_constructor_method(self):
     self._verify_row(build_method.schema, build_method.payload, {})
 
 
+class JavaJarExpansionServiceTest(unittest.TestCase):
+  def test_classpath(self):
```

Comment:
BTW I think we should add more comprehensive unit testing for "JavaJarExpansionService" and "BeamJarExpansionService" (can be a separate PR).

Reply:
Yeah, I agree.